etsd/keys/views.py

```diff
@@ -44,6 +47,34 @@ def get_context_data(self, *args, **kwargs):
 class PublicKeyDetailView(AdminOrAuthorityQsMixin, DetailView):
     model = models.PublicKey
 
+    def post(self, request, *args, **kwargs):
```

Comment:
Please move this functionality to a different view (i.e PublicKeyAcceptRejectFormView) and use form_valid instead of directly overriding `post`.

Reply:
So the two buttons which will be in a POST form will redirect to another view (formView or maybe UpdateView)? Is that correct? Will that view have a blank template and redirect back to the detailView after accepting/rejecting?

Reply:
The `PublicKeyAcceptRejectFormView` will *only* allow post requests, you don't need to display a form. When you submit the accept/reject form the action will be the `PublicKeyAcceptRejectFormView` that will actually handle it and return back to the detail view.